goto_commands.py

```diff
@@ -0,0 +1,136 @@
+import sublime
+import sublime_plugin
+
+from itertools import dropwhile, takewhile
+from .lint import persist
+
+
+"""
+Implement typical Goto Next Previous Error Commands. If you will, add
+the following to your key bindings.
+
+```
+  { "keys": ["ctrl+k", "n"],
+    "command": "sublime_linter_goto_error",
+    "args": {"direction": "next"}
+  },
+  { "keys": ["ctrl+k", "p"],
+    "command": "sublime_linter_goto_error",
+    "args": {"direction": "previous"}
+  },
+```
+
+Supported args:
+
+* `direction`: 'previous' | 'next'
+* `count`: How wide you will jump. Defaults to 1.
+* `wrap`: Set to True, to jump to the top if you're on the last error and
+          vice versa. Defaults to False
+
```

Comment:
I can't jump to the next file with this if I want to, can I?

Reply:
Doesn't seem like it no.

Reply:
That was not specced. You can do this with `F4`, maybe it could be done, that `F4` immediately jumps to the first error of the next file.

The problem with view jumping is, that we yet don't have an ordered, stable list of views to jump to. That's also a problem of the panel. I would prefer if the panel showed the views in a lru way. But I file an issue about that.

Reply:
Indeed, I'd like this to *not* jump to another file. You can use the panel. for that. So as it is now is excellent IMO.

Reply:
Well, I could use F4, but the selection within the panel is not updated if I use the commands added here, so I can only use either entirely (or fall back to using the mouse). I guess the selection updating stuff could be inherited from #752, but that doesn't need to happen right here in this PR.